internal/event/target/webhook.go

```diff
@@ -281,6 +281,16 @@ func NewWebhookTarget(ctx context.Context, id string, args WebhookArgs, loggerOn
 		}
 	}
 
+	// Add :80 or :443 to the endpoint Host since the port is used in isActive() function
+	if _, _, err := net.SplitHostPort(args.Endpoint.Host); err != nil && strings.Contains(err.Error(), "missing port in address") {
+		switch strings.ToLower(args.Endpoint.Scheme) {
+		case "http":
+			args.Endpoint.Host += ":80"
+		case "https":
+			args.Endpoint.Host += ":443"
+		}
+	}
+
```

Comment:
strange why is this not automatically added by httpClient?

```
package main

import "net/http"
import "fmt"

func main() {
	fmt.Println(http.Get("https://www.google.com"))
}
```

Reply:
The issue is more in isActive() in internal/event/target/webhook.go where it is creating a TCP connection witout specifying a port number; I did not want to change isActive() to not be too much distruptive; but I am not sure why we are not doing a simple GET request in isActive() function.

Reply:
I think that change was by @Praveenrajmani - can you tell us ?

Reply:
We were doing an empty `{}` request body POST request. This ended up creating too many empty requests causing some parsing failure at the receiver end. 

We changed to TCP dials because of this.

I think we can have empty `{}` requests only because the TCP dials will not bypass the loadbalancer.

Reply:
https://github.com/minio/minio/commit/443250d1354a7022af76587ea55cbd82470ce1d8